posts/2023/unstructured-grid-collab-1.md

```diff
@@ -0,0 +1,106 @@
+---
+author: Deepak Cherian
+date: 2023-05-05
+tags: tutorial hackathon events
+---
+
+# Unstructured Grid Collaborative Work Time
+
+ESDS hosted our first Collaborative Work Time event on April 17, 2023.
+The topic of the session was "Working With Unstructured Grids".
+Our goal is to encourage cross-lab collaboration and build lasting science-software partnerships.
+
+The event was hybrid with in-person attendees in the Damon Room at the Mesa Lab.
+A lucky overlap with the Improving Scientific Software conference, meant that collaborators from the DOE
+were also able to attend in-person.
+
+```{image} ../images/unstructured-grid-collab-1.jpg
+:alt: Photo of unstructured grids collaborative work time session.
+:width: 70%
+:align: center
+```
+
+## Plan
+
+Participants were asked to fill out a brief survey, and asked to self-organize by adding a one slide description of a project
+to a [presentation](https://docs.google.com/presentation/d/1UDjNMMCyXcHRT6BvX_Lcs4AEoHpNM8LntEfxWOWwwNQ/edit#slide=id.p) in the weeks preceding the event.
+
+At the event, we broke out in to approximately four groups. A recap from each group follows.
+
+## Recap
+
+#### Orhan Eroglu (CISL):
+
+**Topic:** Project [Raijin](https://raijin.ucar.edu/) & [UXarray](https://uxarray.readthedocs.io/)
+
+*What did your group do?*
+
+The UXarray devs had a chance to discuss use cases and get feedback from the community members on UXarray. We branched into subgroups for different work ideas and coded with our users. We noticed a significant bug and fixed that throughout the end of the day.
+
+*Was it a fruitful experience?*
+
+Very much!
+
+*What worked for your group and what didn't work?*
+
+Sitting together with the users and listening to their expectations was a great opportunity! Since we had multiple UXarray folks as facilitators, things worked quite well for us. That wouldn't happen if we didn't have that many helpers (as a side note for the future)
+
+*Would it be useful to redo the  topic again?  What would you do differently the second time around?*
+
+Yes, after some time. We'd add one or two work idea of our own group as well as those came from the community. There were bugs related to
+1. UXarray's MPAS data ingest (attributes not being parsed) with the dual mesh only (see PRs [#274](https://github.com/UXARRAY/uxarray/pull/274) and [#275](https://github.com/UXARRAY/uxarray/pull/275)), and
```

Comment:
I might move the bugs above to the "what did your group do" section since they are first mentioned there.

Reply:
yikes, thanks for noticing definitely didnt put that in the right place